README.md

```diff
@@ -33,7 +33,7 @@ Run the following commands to clone and run it.
 
 ### To clone the repository
 
-git clone https://github.com/Mov305/To-do
+git clone https://github.com/Mov305/Leaderboard
 
 ### To enter Directory
 
```

Comment:
- [ ] OPTIONAL You still have `todo` content in your setup instruction:
<img width="420" alt="Screen Shot 2022-08-23 at 23 50 14" src="https://user-images.githubusercontent.com/49064106/186253532-53a71524-a791-4340-afb1-8fe943305371.png">